podpac/core/coordinates/stacked_coordinates.py

```diff
@@ -627,4 +645,64 @@ def issubset(self, other):
             return all(a.issubset(o) for a, o in zip(acs, ocs))
 
     def simplify(self):
```

Comment:
@mpu-creare This method works in theory, but the calculation is not precise enough for the geotransform to pass an np.allclose test. Either I need some help with it to make it more precise, we need to relax the test, or we need an entirely different way to handle roundtrip of AffineCoordinates -> StackedCoordinates -> AffineCoordinates that can happen during interpolation.